pkgs/test_core/lib/src/runner/vm/platform.dart

```diff
@@ -246,8 +246,14 @@ stderr: ${processResult.stderr}''');
 
   /// Runs [uri] in an isolate, passing [message].
   Future<Isolate> _spawnIsolateWithUri(Uri uri, SendPort message) async {
-    return await Isolate.spawnUri(uri, [], message,
-        packageConfig: await packageConfigUri, checked: true);
+    return await Isolate.spawnUri(
+      uri,
+      [],
+      message,
+      packageConfig: await packageConfigUri,
+      checked: true,
+      debugName: 'test_main',
```

Comment:
Where are these surfaced?

Is there any reason to _not_ put more detail in here? WDYT About

```suggestion
      debugName: 'test_suite:$uri',
```

Reply:
They're surfaced in the VM service API. They're designed to be human readable names and are handy for debugging. So yeah, it definitely makes sense to include the test URI.